src/igvfd/schemas/matrix_file.json

```diff
@@ -0,0 +1,107 @@
+{
+    "title": "Matrix File",
+    "$id": "/profiles/matrix_file.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submission of quantification data in multi-dimension format.",
+    "required": [
+        "award",
+        "lab",
+        "md5sum",
+        "file_format",
+        "file_set",
+        "content_type",
+        "reference_files",
+        "dimension1",
+        "dimension2"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "accession",
+        "alternate_accessions",
+        "aliases",
+        "md5sum"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "file.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/processed_files"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "file.json#/dependentSchemas"
+        ]
+    },
+    "type": "object",
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "content_type": {
+            "comment": "Content Type describes the content of the file.",
+            "enum": [
+                "sparse gene count matrix",
+                "contact matrix",
+                "transcriptome annotations",
+                "sparse transcript count matrix",
+                "sparse peak count matrix"
+            ]
+        },
+        "dimension1": {
+            "comment": "First dimension of the matrix",
+            "type": "string",
+            "enum": [
+                "cell",
+                "gene",
+                "fragment",
+                "treatment",
+                "time",
+                "variant"
+            ]
+        },
+        "dimension2": {
+            "comment": "Second dimension of the matrix",
+            "type": "string",
+            "enum": [
+                "cell",
+                "gene",
+                "fragment",
+                "treatment",
+                "time",
+                "variant"
+            ]
+        },
+        "file_format": {
+            "enum": [
+                "h5ad",
+                "hdf5",
+                "h5ad",
+                "matrixmarket"
```

Comment:
I think the file extension for this is "mtx" so maybe that should be the enum? We may need to double check.

Reply:
We certainly have .mtx.gz files coming from Cell ranger. I also have seen this come in a tar of 3 files.  Specifically these 3 https://support.10xgenomics.com/single-cell-multiome-atac-gex/software/pipelines/latest/output/matrices I am not sure if we are going to accept them merged like that.